src/linear_algebra/alternating.lean

```diff
@@ -207,6 +207,28 @@ instance [distrib_mul_action Sᵐᵒᵖ N] [is_central_scalar S N] :
 
 end has_smul
 
+/-- The cartesian product of two alternating maps, as a multilinear map. -/
+def prod {N'} [add_comm_monoid N'] [module R N'] (f : alternating_map R M N ι)
+  (g : alternating_map R M N' ι) : alternating_map R M (N × N') ι :=
+{ map_eq_zero_of_eq' := λ v i j h hne, prod.ext (f.map_eq_zero_of_eq _ h hne)
+    (g.map_eq_zero_of_eq _ h hne),
+  .. f.to_multilinear_map.prod g.to_multilinear_map }
+
+/-- Combine a family of alternating maps with the same domain and codomains `N i` into an
+alternating map taking values in the space of functions `Π i, N i`. -/
+def pi {ι' : Type*} {N : ι' → Type*} [∀ i, add_comm_monoid (N i)] [∀ i, module R (N i)]
+  (f : ∀ i, alternating_map R M (N i) ι) : alternating_map R M (∀ i, N i) ι :=
+{ map_eq_zero_of_eq' := λ v i j h hne, funext $ λ a, (f a).map_eq_zero_of_eq _ h hne,
+  .. multilinear_map.pi (λ a, (f a).to_multilinear_map) }
```

Comment:
Should these be `simps`?

Reply:
And if `simps` doesn't generate it, can you add the lemmas that say `(a.prod b).to_multilinear_map = a.to_multilinear_map.prod b.to_multilinear_map`?

Reply:
Done, thanks!